### ISSUE TITLE
Permission issue on QNAP NAS during startup (chmod /var/log/gitlab/nginx to ${GITLAB_USER})

### ISSUE THREAD

**Issue by Wyall** [sameersbn/docker-gitlab#1415]
On our QNAP NAS, the installation seems to be working fine, either from their "Container Station" or via docker-compose, but once we try to start one of the three containers, redis and postgresql start, but gitlab_gitlab_1 experiences a permission error: 

Initializing logdir...                                                                                                                            
chown: changing ownership of '/var/log/gitlab/nginx': Disk quota exceeded
Initializing logdir...                                                                                                                            
chown: changing ownership of '/var/log/gitlab/nginx': Disk quota exceeded
...

This happens a few times and then it stops. 

The problems seems to be here: https://github.com/sameersbn/docker-gitlab/blob/fd5343dce2c288f3306b75cd2708838c0c68cc60/assets/runtime/functions#L1161-L1169

Probably this line:   chown -R ${GITLAB_USER}: ${GITLAB_LOG_DIR}/nginx

Do you have any idea what's going wrong?   chown -R root: ${GITLAB_LOG_DIR}/supervisor seems to work fine right before that.

We didn't find a user "git" anywhere (this seems to be the value of ${GITLAB_USER}), but creating one manually with no password didn't work either.
We have ACL activated, this seems to cause issues sometimes, but I'm not sure if it's the case here and how to fix it.

**Comment by Bouvere** [comment]
I think the problem exists already in the install.sh
On QNAP it seems like the **--disabled-login** flag in
https://github.com/sameersbn/docker-gitlab/blob/fd5343dce2c288f3306b75cd2708838c0c68cc60/assets/build/install.sh#L52
does not exist. 
When running only this command I get the error message:
```
adduser: unrecognized option '--disabled-login'
BusyBox v1.24.1 (2017-11-01 02:37:31 CST) multi-call binary.
```
So the user is never created and for sure this will lead to several issues.
I will try to run the container with this flag deleted again. Maybe this was not the only issue.

**Comment by Bouvere** [comment]
Hi there,

found some more information.
I was able to get into the docker (via SSH on my QNAP) by running

`docker run -it sameersbn/gitlab:8.9.6-1 bash`

_8.9.6-1 is the most current version supported by QNAP_

Next I just ran for testing the commands highlighted above

```
mkdir -p ${GITLAB_LOG_DIR}/nginx
chmod -R 0755 ${GITLAB_LOG_DIR}/nginx
chown -R ${GITLAB_USER}: ${GITLAB_LOG_DIR}/nginx
```

First two are working well BUT again the third is throwing the very same error message

`chown: changing ownership of '/var/log/gitlab/nginx': Disk quota exceeded`

I was playing around for a while and the user git is just not allowed to create anything at all on this volume.

Any idea why or how to resolve this?
